### DISCUSSION TITLE
Moodle Tests Musictheory Settings can't be saved

### DISCUSSION THREAD

**Discussion by Mokkaswing** [community/community#37178] [GitHub Education]
Hello,

thank you very much for developping Musictheory question types. They are very useful and I often apply them with students. But now there is a problem:
For practising tests I use the settings "Mehrfachversuche mit Hinweisen" (More tryals with advices?) and allow second editing of the questions. In the question direct I choose the options "Falsche Antworten löschen" and "Anzahl der richtigen Antworten anzeigen" . But if I save these settings, they are gone after. Support team thinks it could be a bug. Please can you repair this? Espacially in Musictheory such as building different keys ect. students need to know exactly wich signs (b or #) are correct and wich not. So this would be very important and helpful.
Thank you for your reply.
Greetings Amely Munz
![Hinweise Test](https://user-images.githubusercontent.com/116624737/197702942-5331f41c-fe12-41a6-9425-702263adef91.png)

**Comment by ossanna16** [comment]
Hi @Mokkaswing,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the specific topic. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
